presto-main/src/main/java/io/prestosql/sql/planner/SymbolsExtractor.java

```diff
@@ -46,6 +49,7 @@ private SymbolsExtractor() {}
     {
         ImmutableSet.Builder<Symbol> uniqueSymbols = ImmutableSet.builder();
         extractExpressions(node).forEach(expression -> uniqueSymbols.addAll(extractUnique(expression)));
+        uniqueSymbols.addAll(extractOrderBySymbols(node, true, noLookup()));
```

Comment:
there are more "hidden" symbols, e.g `io.prestosql.sql.planner.plan.WindowNode.Function`. This boils down again to the fact that correlated symbols can appear out of blue, see https://github.com/prestodb/presto/issues/8459#issuecomment-314372714

Reply:
https://github.com/prestosql/presto/issues/473